tests/impl.cpp

```diff
@@ -350,22 +350,22 @@ result_t do_mm_store_ps(int32_t *p, int32_t x, int32_t y, int32_t z, int32_t w)
 
 float compord(float a, float b)
 {
-    float ret;
-
-    bool isNANA = isnan(a);
-    bool isNANB = isnan(b);
-    ret = (!isNANA && !isNANB) ? ALL_BIT_1_32 : 0.0f;
-    return ret;
+    return (!isnan(a) && !isnan(b)) ? ALL_BIT_1_32 : 0.0f;
 }
 
 double compord(double a, double b)
 {
-    double ret;
+    return (!isnan(a) && !isnan(b)) ? ALL_BIT_1_64 : 0.0f;
+}
 
-    bool isNANA = isnan(a);
-    bool isNANB = isnan(b);
-    ret = (!isNANA && !isNANB) ? ALL_BIT_1_64 : 0.0f;
-    return ret;
+float compunord(float a, float b)
```

Comment:
Shall we revise the function names for readability?

Reply:
Like `cmpunord` or `existNaNOp`?

Reply:
We can introduce the name scheme such as `cmp_` (for comparison) `isNaN` (`NaN` may look better than `NAN`).

Reply:
By the way, this document is useful: https://www.agner.org/optimize/nan_propagation.pdf

Reply:
I will rewrite `compord` as `cmp_noNaN` and `compunord` as `cmp_hasNaN`.